libs/design/tree/src/tree/tree.component.ts

```diff
@@ -154,4 +170,22 @@ export class DaffTreeComponent extends _daffTreeBase implements OnInit, OnChange
       this.flatTree = flattenTree(this._tree, this.renderMode === 'not-in-dom');
     });
   }
+
+  ngAfterViewInit(): void {
+    this.items.changes.pipe(
+      takeUntilDestroyed(this.destroyRef),
+    ).subscribe((items: QueryList<DaffTreeItemDirective>) => {
+      runInInjectionContext(this.injector, () => afterNextRender(() => {
```

Comment:
Is the `selected` state only available after `ngAfterViewInit`?  Can this be done sooner? Perhaps `ngAfterContentInit`?

If this needs `ngAfterViewInit` can we do this without a subscription?

Reply:
the `ContentChildren` isn't available in `ngAfterContentInit` because they're rendered with `ngTemplateOutlet`. yeah I could do it without the sub

Reply:
actually no, this doesn't work. The `selected` value of the tree items isn't updated